src/md4c.c

```diff
@@ -4299,6 +4317,17 @@ md_process_inlines(MD_CTX* ctx, const MD_LINE* lines, int n_lines)
                     const CHAR* dest = STR(opener->end);
                     SZ dest_size = closer->beg - opener->end;
 
+                    MD_SPAN_MENTION_DETAIL det;
+                    if (CH(mark->beg) == '@')
+                    {
+                        det.text = (char *) ctx->text + mark->beg + 1;
+                        det.size = mark->end - mark->beg - 1;
+                        MD_ENTER_SPAN(MD_SPAN_MENTION, &det);
+                        MD_TEXT(text_type, STR(mark->beg), mark->end - mark->beg);
+                        MD_LEAVE_SPAN(MD_SPAN_MENTION, &det);
+                        break;
+                    }
+
```

Comment:
How do you know at this point it is a mention link and a permissive e-mail auto-link? Both extensions may be enabled at the same time.